stdlib/string.gr

```diff
@@ -1425,29 +1409,18 @@ let encodeHelp = (string: String, encoding: Encoding, includeBom: Bool) => {
 }
 
 /**
- * Encodes the given string using the given encoding scheme, excluding any byte-order marker.
+ * Encodes the given string using the given encoding scheme.
  *
  * @param string: The input string
  * @param encoding: The encoding to use
+ * @param includeBom: Whether or not to include a byte order marker (false by default)
```

Comment:
We should add a `@default` attribute to graindoc that allows us to document defaults in plain language to avoid parentheticals like this. I'm not going to block on it though.

Reply:
#1991